### DISCUSSION TITLE
2024/07/28/to-freshman/

### DISCUSSION THREAD

**Discussion by giscus** [stepbystepcode/my-blog#7] [Announcements]
# 2024/07/28/to-freshman/

山西大学作为一所非985、211高校，学校的教学质量和学术氛围相对较弱，可能你的舍友每天会打游戏、看电视剧，可能你的老师会敷衍了事、不负责任，可能你的同学会考试作弊、抄袭论文。 不要等到毕业时才后悔，不要等到工作时才发现自己的能力不足，不要等到社会竞争激烈时才感到无助。所以我们要自救，积极主动地提升自己的学术水平和综合素质，为未来的发展打下坚实的基础。 你可以通过自己的努力和学习，打破这种局限，提

https://blog.stepbystep.host/2024/07/28/to-freshman/

**Comment by kungfudaibi** [comment]
太帅了京哥

**Comment by diyuyizhu** [comment]
给京哥打call！！！
